src/library/Truncate/index.js

```diff
@@ -0,0 +1,2 @@
+/* @flow */
+export { default } from './Truncate';
```

Comment:
Unless this component is a new public component, the git commit should not be marked as a `feat()`, otherwise it will show up as a feature in the changelog.